README.developer.md

```diff
@@ -153,6 +153,26 @@ meson compile -C builddir
 meson test -C builddir
 ```
 
+or simply:
+
```

Comment:
Do we really need to extend Makefile usage when we chose meson as the build tool? Shouldn't we rather add those command using meson?